contrib/cryptomb/private_key_providers/source/cryptomb_stats.cc

```diff
@@ -0,0 +1,26 @@
+#include "contrib/cryptomb/private_key_providers/source/cryptomb_stats.h"
+
+#include "source/common/stats/utility.h"
+
+#include "absl/strings/str_cat.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace PrivateKeyMethodProvider {
+namespace CryptoMb {
+
+CryptoMbStats::CryptoMbStats(Stats::Scope& scope, uint32_t max_queue_size,
+                             absl::string_view stats_prefix,
+                             absl::string_view queue_size_stat_prefix)
+    : stat_name_pool_(scope.symbolTable()), stats_prefix_(stat_name_pool_.add(stats_prefix)) {
```

Comment:
stats_prefix_ is not referenced after the constructor so you can drop the member variable and make this a local.

Reply:
This does not exist anymore after the histogram change.